### PR TITLE
fix(styles): enforce styles for 5.1 release

### PR DESCRIPTION
**Pull request by Skaiir**
Related to #3025

### CODE REVIEW

File: client/src/app/App.js [hunk 1]
```diff
@@ -61,6 +61,7 @@ import History from './History';
 import { PluginsRoot } from './plugins';
 
 import css from './App.less';
+import './App-patch.css';
```
Comment (pinussilvestrus) [inline client/src/app/App.js:64]: Can we add a note here about why we are doing this? So it's easier for us to understand once we remove it. 

As far as I understand this includes

* Making sure element-templates styles are applied
* Sticky headers are disabled

**Comment by Skaiir** [comment]
https://camunda-modeler-on-demand.s3.eu-central-1.amazonaws.com/stylePatchBuild51/camunda-modeler-stylePatchBuild51-linux-x64.tar.gz
    https://camunda-modeler-on-demand.s3.eu-central-1.amazonaws.com/stylePatchBuild51/camunda-modeler-stylePatchBuild51-mac.dmg
    https://camunda-modeler-on-demand.s3.eu-central-1.amazonaws.com/stylePatchBuild51/camunda-modeler-stylePatchBuild51-mac.zip
    https://camunda-modeler-on-demand.s3.eu-central-1.amazonaws.com/stylePatchBuild51/camunda-modeler-stylePatchBuild51-win-ia32.zip
    https://camunda-modeler-on-demand.s3.eu-central-1.amazonaws.com/stylePatchBuild51/camunda-modeler-stylePatchBuild51-win-x64.zip

**Comment by barmac** [comment]
I can see the select template button, there are no sticky headers, and I couldn't see new glitches in the properties panel.

**Comment by nikku** [comment]
@christian-konrad Please review: https://github.com/camunda/camunda-modeler/pull/3026#issuecomment-1181452203.

**Review by pinussilvestrus — APPROVED** [review]

**Pull request merged**